src/widgets/plots/null_freq_plot.py

```diff
@@ -0,0 +1,26 @@
+from PyQt5.QtWidgets import QMessageBox
+from widgets.plots import Plot
+
+
+class NullFrequencyPlot(Plot):
+    def __init__(self, data):
+        super().__init__(data)
+
+    def plot(self):
+        try:
+            ax = self.canvas.figure.subplots()
+            nulls = self.data.value_counts(normalize=True).get('null', 0)
```

Comment:
For when the ""null"" is replaced it should probably look something like that:
```suggestion
            nulls = self.data.isna().sum()/len(self.data)
```